sections/user/edit.php

```diff
@@ -51,6 +51,13 @@
     $Paranoia = array();
 }
 
+// Fetching API Token
+$apiToken = $DB->query("
+    SELECT Token FROM api_applications
```

Comment:
`api_applications` table should add uniq key for user_id

Reply:
I didn't see any delta changes script. There was only a consolidated script file.


> PS: Also the original api from opusnet provides users the ability to generate and keep multiple api keys. Had to restrict that since i don't want to make UI changes as well.